src/clojure/nrepl/server.clj

```diff
@@ -79,6 +79,12 @@
   (t/send transport (response-for msg :status #{:error :unknown-op :done} :op op)))
 
 (def default-middlewares
+  "middleware functions vars that are implicitly merged with any user-specified
+   middleware provided to nrepl.server/default-handler
+
+   For new middleware, use the convention `wrap-` prefix.
```

Comment:
I think this is not the place to mention naming conventions.

Reply:
ok!